sdk-setup/src/mb2

```diff
@@ -2650,6 +2655,8 @@ run_build() {
     local hook_exports=()
     local rpms=$(find_rpms "$OPT_OUTPUTDIR" "$OPT_PKGDIR" "$STATEDIR/spec")
     run_hook post-package "${hook_exports[*]}" ${rpms} || fatal "The 'post-package' hook failed"
+
+    maybe_run_check
```

Comment:
Now rpmlint failure results to non-zero overall exit code. Shouldn't we relax it?

Reply:
I agree, at least for now. Maybe some day when the expectation is that rpmlint doesn't fail we can then make it stricter again.